solutions/standard/variables.tf

```diff
@@ -315,3 +315,25 @@ variable "admin_pass_sm_secret_name" {
   description = "The name of a new elasticsearch administrator secret. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
   default     = "elasticsearch-admin-password"
 }
+
+##############################################################
+# Kibana Configuration
+##############################################################
+
+variable "code_engine_project_name" {
+  description = "Name of the code engine project to deploy the Kibana app. If no value is passed, a new code engine project will be created."
+  type        = string
+  default     = null
+}
+
+variable "enable_kibana_dashboard" {
+  type        = bool
+  description = "Set it true to deploy Kibana dashboard for Elasticsearch instance. NOTE: Kibana image is coming direcly from the official registry (https://www.docker.elastic.co/) and not certified by the IBM."
+  default     = false
+}
+
+variable "es_full_version" {
+  description = "Full version of the Elasticsearch instance in the format `x.x.x` to deploy Kibana dashboard."
```

Comment:
indicate that this is optional, and that the automation will try to look up the version from the elasticsearch apis if this is not specified.